src/UmbracoIdentity/FrontEndCookieAuthenticationOptions.cs

```diff
@@ -8,10 +11,20 @@ namespace UmbracoIdentity
     /// </summary>
     public class FrontEndCookieAuthenticationOptions : CookieAuthenticationOptions
     {
-        public FrontEndCookieAuthenticationOptions(FrontEndCookieManager frontEndCookieManager)
+        public FrontEndCookieAuthenticationOptions(
```

Comment:
This will be a breaking change, you'll need to create new methods and obsolete old methods to ensure that everyone's current installs continue to work.

Reply:
Quite right, I've added the previous constructor back in with an Obsolete attribute. LightInject uses the constructor with the extended parameters primarily too which is a bonus.